src/Features/VisualBasic/Portable/Completion/KeywordRecommenders/Declarations/GetSetKeywordRecommender.vb

```diff
@@ -71,7 +71,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.KeywordRecommenders.Decl
                 recommendations.Add(New RecommendedKeyword("Set", VBFeaturesResources.Declares_a_Set_property_procedure_that_is_used_to_assign_a_value_to_a_property))
             End If
 
-            Return recommendations
+            Return recommendations.ToImmutableArray()
```

Comment:
💡 This is a good candidate for `TemporaryArray(Of RecommendedKeyword)` (always 0-2 elements)